app/helpers/form_helper.rb

```diff
@@ -192,6 +192,18 @@ def autocomplete_f(f, attr, options = {})
     end
   end
 
+  def byte_size_f(f, attr, options = {})
+    options[:class] = options[:class].to_s + ' byte_spinner'
+    options[:help_inline] ||= popover('', _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified."))
+    options[:help_block] ||= content_tag(:span, :class => 'maximum-limit hidden') do
+      content_tag(:i, '', :class => 'pficon-warning-triangle-o') +
+      content_tag(:span, ' ' + _('Specified value is higher than recommended maximum'), :class => 'error-message')
+    end
+    options[:help_block] += f.hidden_field(attr, :class => "real-hidden-value", :id => "#{f.object.object_id}_hidden_real_value")
```

Comment:
I'd prefer not having an id (`:id => nil`) here unless there is any reason for it. `f.object` may have multiple byte size fields.